internal/app/configupdates.go

```diff
@@ -151,11 +151,15 @@ func (processor *ConfigUpdateProcessor) processConfigChangedPipeline() {
 		sdk.runtime.TargetType = sdk.targetType
 
 		// Update the pipelines with their new transforms
-		for _, pipeline := range pipelines {
-			// TODO: Look at better way to apply pipeline updates
-			sdk.runtime.SetFunctionsPipelineTransforms(pipeline.Id, pipeline.Transforms)
-			sdk.runtime.SetFunctionsPipelineTopics(pipeline.Id, pipeline.Topics)
+		for i, pipline := range pipelines {
+			var fullTopics []string
+			for _, topic := range pipline.Topics {
+				fullTopics = append(fullTopics, coreCommon.BuildTopic(sdk.config.MessageBus.GetBaseTopicPrefix(), topic))
+			}
```

Comment:
I noticed that the logic here is very similar to what the [getFullTopics](https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/internal/app/service.go#L408-L422) func already does. To avoid duplicating logic, would it be possible to refactor this to call `getFullTopics` instead?